share/completions/brew.fish

```diff
@@ -6,10 +11,32 @@ function __fish_brew_needs_command
   return 1
 end
 
+function __fish_brew_needs_services_action
+  __fish_brew_using_command services
+
+  set cmd (commandline -opc)
+  if test (count $cmd) -le 2
+    return 0
+  end
+  return 1
+end
+
+function __fish_brew_services
+  brew services list | awk '{if (NR>1) print $1}'
+end
+
 function __fish_brew_using_command
   set cmd (commandline -opc)
-  if [ (count $cmd) -gt 1 ]
-    if [ $argv[1] = $cmd[2] ]
+  set index 2
+
+  if [ (count $argv) -gt 1 ]
+    if test -n "$argv[2]"
+      set index $argv[2]
+    end
+  end
+
+  if [ (count $cmd) -ge $index ]
```

Comment:
sorry. I got a little confused. using `-q` option error `Array index out of bounds`
still can occurs.

```
$ set -l sss a b c d e f
$ set -q $sss[7]
Array index out of bounds
fish: set -q $sss[7]
                  ^
```